aspnetcore/grpc/aspnetcore.md

```diff
@@ -77,7 +95,63 @@ The preceding Windows 10 Build versions may require the use of a [Windows Inside
 
 For more information about choosing the right server for an ASP.NET Core app, see <xref:fundamentals/servers/index>.
 
-:::moniker range=">= aspnetcore-5.0"
+:::moniker range=">= aspnetcore-6.0"
+
+## Kestrel
+
+[Kestrel](xref:fundamentals/servers/kestrel) is a cross-platform web server for ASP.NET Core. Kestrel provides the best performance and memory utilization, but it doesn't have some of the advanced features in HTTP.sys such as port sharing.
```

Comment:
```suggestion
[Kestrel](xref:fundamentals/servers/kestrel) is a cross-platform web server for ASP.NET Core. Kestrel provides the optimal performance and memory utilization, but it doesn't have some of the advanced features in HTTP.sys such as port sharing.
```

Reply:
We usually get called out on marketing phrases where we say something "provides the best" or "is the best".   However, it is key to your point.  I gave it a shot but feel free to skip my suggestion here.